tests/testthat/test-fct_lump.R

```diff
@@ -128,6 +128,14 @@ test_that("fct_lump_min works when weighted", {
   expect_equal(levels(fct_lump_min(f, min = 1.5, w = w)), c("b", "c", "d", "Other"))
 })
 
+test_that("throws error if n or prop is not numeric", {
+  f <- c("a", "a", "a", "a", "b", "b", "b", "c", "c", "d")
+
+  expect_error(fct_lump(f, n = "2"))
```

Comment:
Can you please add a "n" and "prop" to the test, just to match sure that you're catching the error that you think you are?

Reply:
done